src/storage/mod.rs

```diff
@@ -1714,7 +1708,7 @@ impl<E: Engine, L: LockManager, Api: APIVersion> Storage<E, L, Api> {
                     let mut stats = Statistics::default();
                     let result: Vec<Result<KvPair>> = keys
                         .into_iter()
-                        .map(Key::from_encoded)
+                        .map(|k| Api::encode_raw_key_owned(k, None))
```

Comment:
Why not move it to following map?

Reply:
En...add a method, e.g. "generate_delete_modify(keys: Vec<Vec<u8>>)", to trait [`APIVersion`](https://github.com/tikv/tikv/blob/45fcb0812a0437cabce2436c5223c96c9de83d5e/components/api_version/src/lib.rs#L15) ?

Reply:
Fixed, PTAL~